ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -65,13 +83,14 @@ class ConfirmToProceedTransactionScreen extends React.Component<Props, never> {
    * of the order of of higher 10^13
    *  @https://www.pivotaltracker.com/n/projects/2048617/stories/157769657
    */
-  private getTotalAmount(transaction: Readonly<WalletTransaction>) {
-    return transaction.amount + transaction.transactionCost;
+  private getTotalAmount() {
+    return (
+      AmountInEuroCentsFromNumber.encode(this.props.amount) +
```

Comment:
`AmountInEuroCentsFromNumber.encode(this.props.amount + this.props.fee)` ?

Reply:
Under the hood, `AmountInEuroCents` is a string, so the `+` operator would concatenate the two